src/elements/content-sidebar/SidebarUtils.js

```diff
@@ -151,6 +152,8 @@ class SidebarUtils {
                 return <FormattedMessage {...messages.sidebarMetadataTitle} />;
             case SIDEBAR_VIEW_ACTIVITY:
                 return <FormattedMessage {...messages.sidebarActivityTitle} />;
+            case SIDEBAR_VIEW_DOCGEN:
```

Comment:
Do we have tests for this in the SidebarUtils test file? In general let's make sure that we have tests for all new functionality we've added.